src/framework/global/types/string.cpp

```diff
@@ -110,6 +110,7 @@ char16_t Char::fromAscii(char c)
 
 char Char::toAscii(bool* ok) const
 {
+    UNUSED(ok);
```

Comment:
fixed here https://github.com/musescore/MuseScore/pull/11967
(`ok` needs to be pass to `toAscii(m_ch)`)